src/App.svelte

```diff
@@ -0,0 +1,216 @@
+<script>
+  import { db } from "./firebase.js";
+  import { onMount } from "svelte";
+  import AddItemForm from "./AddItemForm.svelte";
+  import GroceryItem from "./components/GroceryItem.svelte";
+
+  let name;
+  let newItemList = {};
+
+  let groceryList = [];
+  let shareIconImage = "./images/share-icon.svg";
+  let bellIconImage = "./images/bell-regular.svg";
+  let redoIconImage = "./images/redo-solid.svg";
+  let noteIconImage = "./images/sticky-note-regular.svg";
+  let calendarIconImage = "./images/calendar.svg";
+  let dateString;
+  let isEditing = false;
+  let xxx = "";
+
+  let now = new Date(),
+    month,
+    day,
+    year;
+
+  onMount(() => {
+    db.collection("grocery").onSnapshot((snapData) => {
+      groceryList = snapData.docs;
+    });
+    (month = "" + (now.getMonth() + 1)),
+      (day = "" + now.getDate()),
+      (year = now.getFullYear());
+    if (month.length < 2) month = "0" + month;
+    if (day.length < 2) day = "0" + day;
+    dateString = [year, month, day].join("-");
+  });
+
+  function removeItem(itemId) {
+    console.log("The event works");
+    console.log(itemId);
+    db.collection("grocery").doc(itemId).delete();
+  }
+
+  // async function updateItem(itemId, newItemValue) {
+  //    const response = await db.collection("grocery").doc(itemId).update({ name: newItemList[itemId] });
+  //   newItemList[itemId] = '';
+  //   isEditing = false;
+
+  // }
+
+  function editItem() {
+    isEditing = true;
+  }
+
+  function updateItem(itemId, inputEl) {
+    console.log(itemId);
+    console.log(inputEl);
+    console.log(newItemList);
+    db.collection("grocery").doc(itemId).update({ name: inputEl });
+    console.log("no update");
+    isEditing = false;
+
+    // console.log(isEditing);
+    // isEditing = false;
+  }
+</script>
+
+<header />
+<main>
+  <h1>Grocery list</h1>
+  <h3>
```

Comment:
This might look like a heading, but it's not, semantically speaking, a heading. It can be a span with CSS.

Reference: https://accessibility.umn.edu/what-you-can-do/start-7-core-skills/headings

Reply:
Thanks for the video, that was very interesting. 
To be sure that I got your point, in this case It's more relevant to put a <span> tag instead of an <header> tag and even if I'd still add this header tag I should do it differently. My question is how? Did you mean that It doesn't make sense to put a header tag with nothing in it?

Reply:
Or maybe you meant that header stylings have to be writing first in the <style> section?

Reply:
What I mean is that it doesn't make sense the date to be a h3 because it doesn't hold that purpose. it's just a simple date, not a type of heading. imagine this website was a book, a recipe book, the date wouldn't be a title, right?

so what I meant is to just replace it with a span.

here's another resource, a short video, about headings: https://www.youtube.com/watch?v=vAAzdi1xuUY

Reply:
Got it!